packages/driver/src/cypress/error_messages.ts

```diff
@@ -1381,6 +1381,10 @@ export default {
   },
 
   select: {
+    no_argument: {
+      message: `${cmd('select')} was not called with any value, index, or text.`,
+      docsUrl: 'https://on.cypress.io/select',
+    },
```

Comment:
Let's update this to be called `invalid_argument` and make it similar to [this error message](https://github.com/cypress-io/cypress/blob/26f92e0e56d59cfa83207d101ad95683744f3696/packages/driver/src/cypress/error_messages.ts#L420-L423), so it details which types are support (string, number, or array) and shows what value the user passed in.